python/ray/tune/experimental/output.py

```diff
@@ -110,22 +110,28 @@ def _get_time_str(start_time: float, current_time: float) -> Tuple[str, str]:
     delta: datetime.timedelta = current_time_dt - start_time_dt
 
     rest = delta.total_seconds()
-    days = rest // (60 * 60 * 24)
+    days = int(rest // (60 * 60 * 24))
 
     rest -= days * (60 * 60 * 24)
-    hours = rest // (60 * 60)
+    hours = int(rest // (60 * 60))
 
     rest -= hours * (60 * 60)
-    minutes = rest // 60
+    minutes = int(rest // 60)
 
-    seconds = rest - minutes * 60
+    seconds = int(rest - minutes * 60)
 
     if days > 0:
-        running_for_str = f"{days:.0f} days, "
+        running_for_str = f"{days:d}d "
     else:
         running_for_str = ""
 
-    running_for_str += f"{hours:02.0f}:{minutes:02.0f}:{seconds:05.2f}"
+    if hours > 0 or days > 0:
+        running_for_str += f"{hours:d}hr "
+
+    if minutes > 0 or hours > 0 or days > 0:
+        running_for_str += f"{minutes:d}min "
```

Comment:
would it be simpler to just check whether running_for_str already has some content? like
```
running_for_str = ""
if days > 0:
    running_for_str += f"{days:d}d "
if hours > 0 or running_for_str:
    running_for_str += f"{hours:d}hr "
if minutes > 0 or running_for_str:
    running_for_str += f"{minutes:d}min "
running_for_str += f"{seconds:d}s"
```

Reply:
Sure, good idea. Updated!